src/supply_chain/policies.py

```diff
@@ -0,0 +1,186 @@
+from abc import ABCMeta, abstractmethod
+from GA_core.gene import Gene
+
+
+class PolicyInterface(Gene, metaclass=ABCMeta):
```

Comment:
no need for inheritence here

This is the abstract definition.

Genes are the building blocks for the solution. A/the solution is not a collection of policies, rather than a collection of genes.

The gene class should either inherite or injected with the the policy

Reply:
I thought a policy is a gene?

The inheritance of the `Gene` class was intentional here, to keep the GA code decoupled from the SC code (in case we want to reuse the GA outside the SC scope in the future, the Gene class ensures that all the potential genes have the required methods).